aospy/test/test_calc_basic.py

```diff
@@ -160,6 +161,53 @@ def setUp(self):
         }
 
 
+test_params = {
+        'proj': example_proj,
+        'model': example_model,
+        'run': example_run,
+        'var': precip_largescale,
+        'date_range': 'default',
+        'intvl_out': 1,
+}
+
+
+@pytest.mark.parametrize(
+    ('dtype_out_time'),
+    [(None),
+     ('None'),
+     ([])]
+)
+def test_calc_object_no_time_options(dtype_out_time):
```

Comment:
This test doesn't assert anything; it will only fail if the `CalcInterface` call fails.  So let's add an assertion about the `dtype_out_time` in the resulting `CalcInterface` object.